EMSMobileSDK/EMSMobileSDK.swift

```diff
@@ -102,6 +103,23 @@ public class EMSMobileSDK: NSObject {
     */
     @objc
     public func remoteNotificationReceived(userInfo: [AnyHashable: Any]?) {
+        if UserDefaults.standard.string(forKey: "Initialize") != nil,
+           UIApplication.shared.applicationState == .inactive {
+            UserDefaults.standard.removeObject(forKey: "Initialize")
+            return
+        }
+        
+        guard let openUrl = userInfo?["ems_open"] as? String else { return }
+        log("Received EMS_OPEN: " + openUrl)
+        
+        apiService.logEMSOpen(url: openUrl) { [weak self] (response) in
+            guard response?.statusCode == 200 else { return }
+            self?.log("Content URL Sent Successfully")
```

Comment:
This 3 lines is duplicated in `initLogEMSOpen`, it would probably be better to create a separate method.